README.md

```diff
@@ -33,7 +33,7 @@ where:
         -d/--dist-dir-path: The path where resulting Agent Plugin archive will be stored.
         Default: <current_working_directory>/dist
 
-        -s/--source-dir: The name of the source directory.
+        -s/--source-dir-name: The name of the source directory.
```

Comment:
We have `--dist-dir-path` and `--source-dir-name`. Do these take different kinds of input? If not, we should shorten them to `--dist-dir` and `--source-dir`.

Reply:
Yep they do. `--dist-dir-path` gets path and `--source-dir-name` a string